website/content/en/docs/best-practices/multi-tenancy.md

```diff
@@ -0,0 +1,162 @@
+
+- **MUST** declare resource requests for both CPU and memory.
+- **OPTIONALLY** setting the resources limit for Memory and CPU (Note that if your project is integrated with OLM then, admins would be able to customize it via the [subscription][olm-subscriptions]). Also, be aware that our project was built with Kubebuilder or SDK scaffold then for the `Operator/manager`(See `config/manager/manager.yaml`), then some values which allow you to get started are scaffolded by default, however, you ought to optimize based on the needs.
+- **Should** provide the mechanisms for [Monitoring compute & memory resource usage][k8s-metrics]. Kubebuilder and SDK provide some [metrics][kb-metrics] which can help you with. Note that cluster admins can use it to monitor and resize the Operator and its Operands.
+- **OPTIONALLY** ensuring that the Operators uses [liveness health check][k8s-probes] which can be used to check the `CPU` performance. If your project was built with Kubebuilder or SDK scaffold then, it is scaffolded by default for the `Operator/manager`. (See `config/manager/manager.yaml`).
+- **Should** allow admins to customize the `requests/limits` amounts defined for the Pod/Deployment Operands created by the Operator and not keep them fixed on the code.
+- **Should** document how cluster admins can customize/rightsize the resources limits for the Operator and Operands.
+
+You also might consider providing solutions that allow vertical pod autoscaler to manage arbitrary scaling for your Operators. [More info][k8s-vertical-pod-scaling]. 
+And then, you also might look for allowing [horizontal pod autoscaling][k8s-autoscaling] for the Operands. In this case, see that resource requests are also needed for horizontal pod autoscaling to work.
+
+**NOTE:** If you are using the k8s OpenShift vendor you might want to check the doc [Automatically adjust pod resource levels with the vertical pod autoscaler
+][ocp-pod-scaling].
+
+### But, what happens
+
+#### when the resource requests are not set?
+
+**If not set:** 
+- configurations made by the cluster administrators such as [ResourceQuota][k8s-resources-quotas] might not work. 
+- the Operators consumers might face resource shortage on a node when resource usage later increases, for example, during a daily peak in request rate.
+- the Operators consumers might install the Operator in a node/namespace which does not have the minimal resources available.
+
+#### when the resource limits are not set?
+
+**If not set:** wrong configurations or code implementations can consume all resources available, affecting other components on cluster. 
+Also, it might let the containers more vulnerable such as to Dos Attacks. [More info][docker-security].
+
+However, note that in this case, as described above, Cluster Admins has popular practices to ensure that all `Pods/Deployments` 
+in a node/namespace have the resources limited such as to use the [ResourceQuota][k8s-resources-quotas] 
```

Comment:
However, note that in this case, as described above, Cluster Admins has popular practices to ensure that all `Pods/Deployments` 
in a node/namespace -. That means **many** configurations which can be applied on the node and/or namespace. 

> such as == example

then, 
> to use the [ResourceQuota][k8s-resources-quotas] 
to configure the total amount of resources that can be consumed which if used in conjunction with namespaces, 
can prevent hitting the usage in one namespace affecting what is deployed on the others.

It is a copy and past from Kubernetes docs, see: 

> The resource quota feature can be configured to limit the total amount of resources that can be consumed. If used in conjunction with namespaces, it can prevent one team from hogging all the resources.
From: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/

Reply:
I tried to improve the text but that seems sorted out.
Please, feel free to ping me if not.